src/test/regress/expected/shard_rebalancer.out

```diff
@@ -1585,8 +1582,8 @@ WHERE table_schema = 'public'
 ) a ORDER BY table_name;
  table_schema | table_name | row_estimate | total_bytes
 ---------------------------------------------------------------------
- public       | tab_123041 |        10000 |      368640
- public       | tab_123043 |        10000 |      368640
+ public       | tab_123041 |        10000 |      xxxxxxx
+ public       | tab_123043 |        10000 |      xxxxxxx
```

Comment:
These sizes are important for the intent of these tests. I'm guessing you changed this to avoid test flakyness? If so there's another way of making this not flaky, that will keep the intent of the test. It's documented here: https://github.com/citusdata/citus/blob/main/src/test/regress/flaky_tests.md#disk-size-numbers-are-not-exactly-like-expected